### DISCUSSION TITLE
Is there a way to use the runner images for setting up self hosted runners?

### DISCUSSION THREAD

**Discussion by StephenHodgson** [community/community#160514] [Actions]
I'd like to setup my self hosted runners based on the runner images and definitions found in this repository, and to keep my images up to date. Is there a way to utilize the image definitions found here to do so?

**Comment by skoonin** [comment]
yup just use it as your base image: 
```
FROM --platform=${OS}/${ARCH} ghcr.io/actions/actions-runner:latest
```

**Reply by StephenHodgson** [reply]
ok, but without context I have no idea how to use this 😅

**Reply by david-burcuta** [reply]
hey @StephenHodgson, did you manage to find a way? I am looking to achieve the same thing 😅

**Comment by crohr** [comment]
@StephenHodgson where are you running your runners? If using AWS, you might be interested in https://github.com/runs-on/runner-images-for-aws, which provides base AMIs for EC2 instances.

**Reply by StephenHodgson** [reply]
Thanks @crohr but unfortunately using a different cloud provider

**Reply by melyux** [reply]
What if you're running a macOS runner on a physical Mac?
